### PR TITLE
SONARJAVA-4514 Update Custom rules documentation to latest LTS SonarQ…

### PR DESCRIPTION
**Pull request by irina-batinic-sonarsource**

### CODE REVIEW

File: docs/CUSTOM_RULES_101.md [hunk 1]
```diff
@@ -553,63 +635,75 @@ $ mvn clean install -f pom_SQ_8_9_LTS.xml
 [INFO] ------------------------------------------------------------------------
 ```
 
-Then, grab the jar file `java-custom-rules-example-1.0.0-SNAPSHOT.jar` from the `target` folder of the project, and move it to the extensions folder of your SonarQube instance, which will be located at `$SONAR_HOME/extensions/plugins`.
+Then, grab the jar file `java-custom-rules-example-1.0.0-SNAPSHOT.jar` from the `target` folder of the project, and move it to the extensions folder of your SonarQube instance, which will be located at `$SONAR_HOME/lib/extensions`.
 
 >
-> :exclamation: **SonarQube Java Plugin compatible version**
+> Exclamation: **SonarQube Java Plugin compatible version**
```
Comment (alban-auzeill) [inline docs/CUSTOM_RULES_101.md:641]: It's not the word `exclamation` surrounded by `:` but the emoji :exclamation:, emojis are a small images used by young people to express an idea or an emotion :trollface:
```suggestion
> :exclamation: **SonarQube Java Plugin compatible version**
```

File: docs/CUSTOM_RULES_101.md [hunk 2]
```diff
@@ -553,63 +635,75 @@ $ mvn clean install -f pom_SQ_8_9_LTS.xml
...
+From there, under the language section, select "**Java**", and then "**The Fellowship's custom rules**" (or "**MyCompany Custom Repository**" if you did not change it) under the repository section. 
+Your rule should now be visible (with all the other sample rules).
 
 ![Selected rules](resources/rules_selected.png)
 
-Once activated (not sure how? see [quality-profiles](https://docs.sonarqube.org/latest/instance-administration/quality-profiles/)), the only step remaining is to analyze one of your project!
+Once activated (not sure how? see [quality-profiles](https://docs.sonarqube.org/latest/instance-administration/quality-profiles/)), the only step remaining is to analyze one of your projects!
 
-When encountering a method returning the same type as its parameter, the issue will now raise issue, as visible in the following picture:
+When encountering a method returning the same type as its parameter, the issue will now raise an issue, as visible in the following picture:
```
Comment (alban-auzeill) [inline docs/CUSTOM_RULES_101.md:660]:
```suggestion
When encountering a method returning the same type as its parameter, the custom rule will now raise an issue, as visible in the following picture:
```

**Review by alban-auzeill — CHANGES_REQUESTED** [review]

File: docs/CUSTOM_RULES_101.md [hunk 3]
```diff
@@ -553,63 +635,75 @@ $ mvn clean install -f pom_SQ_8_9_LTS.xml
 [INFO] ------------------------------------------------------------------------
 ```
 
-Then, grab the jar file `java-custom-rules-example-1.0.0-SNAPSHOT.jar` from the `target` folder of the project, and move it to the extensions folder of your SonarQube instance, which will be located at `$SONAR_HOME/extensions/plugins`.
+Then, grab the jar file `java-custom-rules-example-1.0.0-SNAPSHOT.jar` from the `target` folder of the project, and move it to the extensions folder of your SonarQube instance, which will be located at `$SONAR_HOME/lib/extensions`.
```
[the remark this answers is not in the document]
Comment (irina-batinic-sonarsource) [reply docs/CUSTOM_RULES_101.md:638]: Is this correct?
I think the new path where plugins are located is `$SONAR_HOME/lib/extensions`.

File: docs/CUSTOM_RULES_101.md [hunk 4]
```diff
@@ -522,21 +602,23 @@ public class MyJavaRulesDefinition implements RulesDefinition {
 ## Testing a custom plugin
 
 >
-> :exclamation: **Prerequisite**
-> 
-> For this chapter, you will need a local instance of SonarQube. If you don't have a SonarQube platform installed on your machine, now is time to download its latest version from [HERE](https://www.sonarqube.org/downloads/)!
+> Exclamation: **Prerequisite**
```
Comment (alban-auzeill) [inline docs/CUSTOM_RULES_101.md:605]:
```suggestion
> :exclamation: **Prerequisite**
```

**Review by alban-auzeill — APPROVED** [review]

**Pull request merged**